docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -181,13 +220,13 @@ At the root of your project, run `npm run build plugin` to start the TypeScript
 }
 ```
 
-Now when running `npx expo prebuild` inside our **example** folder we should see our 'my custom plugin' console.log statement in the terminal.
+When you run `npx expo prebuild` inside your **example** folder, you see your 'my custom plugin' console.log statement in the terminal.
```

Comment:
```suggestion
When you run `npx expo prebuild` command inside your **example** directory, you will see "my custom plugin" logged by console statement in the terminal.
```

- Directory over folder as per our writing style guide.
- No single quotes. Let's stick to one pattern (we use double quotes) for concistency.